socorro/external/boto/crashstorage.py

```diff
@@ -15,17 +16,25 @@
     MemoryDumpsMapping,
 )
 from socorro.external.es.super_search_fields import SuperSearchFieldsData
-from socorro.lib.transaction import retry
+from socorro.lib.util import retry
 from socorro.schemas import CRASH_REPORT_JSON_SCHEMA
 
 
-class BotoCrashStorage(CrashStorageBase):
+LOGGER = logging.getLogger(__name__)
+
+
+def wait_time_generator():
+    for i in [1, 1, 1, 1, 1]:
+        yield i
+
+
+class BotoS3CrashStorage(CrashStorageBase):
```

Comment:
`BotoS3CrashStorage` was the only subclass of `BotoCrashStorage`, so I merged the two.